src/Sulu/Bundle/ContactBundle/Resources/js/index.js

```diff
@@ -245,12 +256,10 @@ initializer.addUpdateConfigHook('sulu_contact', (config: Object, initialized: bo
                         iban: {
                             label: translate('sulu_contact.iban'),
                             required: true,
-                            colSpan: 8,
                             type: 'iban',
                         },
                         bic: {
                             label: translate('sulu_contact.bic'),
-                            colSpan: 4,
```

Comment:
With `colSpan: 4`, the field is not wide enough to properly display the field error, because the error is too long